src/components/Login/SelectLogin.tsx

```diff
@@ -37,13 +51,16 @@ export default function SelectLogin(props: Props) {
       method: "post",
       requestData: codeResponse,
     });
+    const googleLoginRes: GoogleLoginRes = data;
     try {
-      if (data) {
+      if (!googleLoginRes.needConnection) {
         // Store user data into local storage after logging
         localStorage.setItem("UserInfo", JSON.stringify(data));
```

Comment:
use "googleLoginRes" variable？

Reply:
> use "googleLoginRes" variable？

Since the type of response is AxiosResponse<any, any>, I tried to use feature of TS to handle response.data, so I assigned it to googleLoginRes.

Reply:
Yes I know, I think you should replace the "data" by the "googleLoginRes" variable to do the localStorage setItem at line 58 ~ line 60.

Reply:
> Yes I know, I think you should replace the "data" by the "googleLoginRes" variable to do the localStorage setItem at line 58 ~ line 60.

Got it! :)